listenbrainz/db/stats.py

```diff
@@ -25,10 +25,9 @@
 import ujson
 
 from listenbrainz import db
-from listenbrainz import config
+from listenbrainz import config 
 
-
-def insert_user_stats(user_id, artists, recordings, releases, artist_count):
+def insert_user_stats(user_id, artists, recordings, releases, artist_count, timestamp):
```

Comment:
`timestamp` isn't a good name here - traditionally I would expect this to refer to a unix timestamp (number of seconds since Jan 1 1970), but it looks like you're using it to mean `yyyy.mm` instead. Maybe call it `yearmonth` instead?

Reply:
yearmonth sounds good. It basically tells us the month and year in which the stats were calculated.
Maybe YearMonth?

Reply:
in python we never name variables with upper case letters.

Reply:
I thought yearmonth lacks distinction. But yeah sure :) Pythonic way is better.